keras_nlp/samplers/beam_sampler.py

```diff
@@ -36,11 +36,14 @@ class BeamSampler(Sampler):
     Args:
         num_beams: int. The number of beams that should be kept at each
             time-step. `num_beams` should be strictly positive.
+        return_all_beams: bool. When set to `True`, the sampler will return the top prompt,
+            all prompts and their respective probabilities score.
 
     Call Args:
         {{call_args}}
 
     Examples:
+    1. Return only the beam with the highest accumulated probability.
```

Comment:
We can remove the number here.

Reply:
Hey @chenmoneygithub,
Made the required changes. Do let me know if there are any other changes we should make. Thanks!